src/components/Dropdown/components/Label/index.tsx

```diff
@@ -65,6 +65,7 @@ const Label = ({
 
   return (
     <div className={styles.itemRowContainer}>
+      {icon ? icon : null}
```

Comment:
Why not directly {icon} ?

```suggestion
      {icon || null}
```